huaweicloud/services/acceptance/drs/resource_huaweicloud_drs_job_test.go

```diff
@@ -25,6 +25,7 @@ func TestAccResourceDrsJob_basic(t *testing.T) {
 	var obj jobs.BatchCreateJobReq
 	resourceName := "huaweicloud_drs_job.test"
 	name := acceptance.RandomAccResourceName()
+	dbName := acceptance.RandomAccResourceName()
```

Comment:
what's the root cause of this issue, you already added an index to the source/destination rds names to make it different now, why should we also need to add a random dbName.